dom0/sd-workstation.top

```diff
@@ -33,12 +38,12 @@ base:
     - sd-sys-firewall-files
   sd-whonix:
     - sd-whonix-hidserv-key
-  sd-log-buster-template:
-    - sd-log-template-files
   securedrop-workstation-buster:
     - sd-workstation-template-files
   sys-usb:
     - sd-usb-autoattach-add
+  whonix-gw-15:
```

Comment:
Why do we apply sd-whonix-template files to whonix-gw-15? shouldn't we apply it to `sd-whonix`?

Reply:
@emkll I can not find any `sd-whonix-template`, the `sd-whonix` uses `whonix-gw-15` and that is why I am applying files to that template.